libs/ngx-gauge/src/gauge/gauge.ts

```diff
@@ -270,15 +275,21 @@ export class NgxGauge implements AfterViewInit, OnChanges, OnDestroy {
             }
         }
 
-        self._animationRequestID = window.requestAnimationFrame((timestamp) => {
-            startTime = timestamp || new Date().getTime();
-            animate(timestamp);
-        });
+        if(animation){
+            self._animationRequestID = window.requestAnimationFrame((timestamp) => {
+                startTime = timestamp || new Date().getTime();
+                animate(timestamp);
+            });
+        } else {
```

Comment:
I am also thinking of similar logic and avoid calling requestAnimationFrame if animation property is false. But I dont understand why you still using previousProgress and ov in else block of the code?